app/controllers/internal/organizations_controller.rb

```diff
@@ -0,0 +1,18 @@
+class Internal::OrganizationsController < Internal::ApplicationController
+  layout "internal"
+
+  def index
+    @organizations = Organization.order("name DESC").page(params[:page]).per(50)
+
+    return if params[:search].blank?
+
+    @organizations = @organization.where(
+      "name ILIKE ?",
+      "%#{params[:search].strip}%",
+    )
+  end
+
+  def show
+    @organization = Organization.find(params[:id])
```

Comment:
In the future if we do more than count the related models we should think about eager loading them here but for now I think this is fine.

Reply:
I agree. Honestly, I'm not really clear on what type of analytics will be helpful here, but we should keep that in mind if Michael or Peter come back with an ask for more data.